Source/SourceKittenFramework/StringView.swift

```diff
@@ -156,13 +158,13 @@ public struct StringView {
 
      - returns: An equivalent `NSRange`.
      */
-    public func byteRangeToNSRange(start: Int, length: Int) -> NSRange? {
```

Comment:
Also, it seems, that for backward better backward compatibility it is a good ideat o leave

```
byteRangeToNSRange(start: ByteCount, length: ByteCount)
```

Reply:
In this case, since the function name hasn't changed, the compiler error should be enough to let users know how to migrate their code.